packages/render/src/operations/transform.js

```diff
@@ -2,113 +2,133 @@ import * as R from 'ramda';
 
 // TODO: Implement using only matrices to support skew and even more operations than css.
 
-const getRotation = transform => {
-  const match = /rotate\((-?\d+(.\d+)?)(.+)\)/g.exec(transform);
-
-  if (match && match[1] && match[3]) {
-    const value = match[1];
-    return match[3] === 'rad' ? (value * 180) / Math.PI : value;
+const parse = transformString => {
+  const transforms = transformString.trim().split(/\) |\)/);
+  // Handle "initial", "inherit", "unset".
+  if (transforms.length === 1) {
+    return [[transforms[0], true]];
   }
 
-  return 0;
-};
-
-const getTranslateX = transform => {
-  const matchX = /translateX\((-?\d+\.?d*)\)/g.exec(transform);
-  const matchGeneric = /translate\((-?\d+\.?d*).*(,|\s)\s*(-?\d+\.?d*).*\)/g.exec(
-    transform,
-  );
-
-  if (matchX && matchX[1]) return matchX[1];
-  if (matchGeneric && matchGeneric[1]) return matchGeneric[1];
+  const parsed = [];
 
-  return 0;
-};
-
-const getTranslateY = transform => {
-  const matchY = /translateY\((-?\d+\.?\d*)\)/g.exec(transform);
-  const matchGeneric = /translate\((-?\d+\.?\d*).*(,|\s)\s*(-?\d+\.?\d*).*\)/g.exec(
-    transform,
-  );
+  for (let i = 0; i < transforms.length; i += 1) {
+    const transform = transforms[i];
 
-  if (matchY && matchY[1]) return matchY[1];
-  if (matchGeneric && matchGeneric[3]) return matchGeneric[3];
+    if (transform) {
+      const [operationName, operationValueRaw] = transform.split('(');
+      const values = operationValueRaw.split(',').map(val => val.trim());
+      const value = values.length === 1 ? values[0] : values;
+      parsed.push([operationName, value]);
+    }
+  }
 
-  return 0;
+  return parsed;
 };
 
-const getScaleX = transform => {
-  const matchX = /scaleX\((-?\d+\.?\d*)\)/g.exec(transform);
-  const matchGeneric = /scale\((-?\d+\.?\d*).*,?\s*(-?\d+\.?\d*).*\)/g.exec(
-    transform,
-  );
-
-  if (matchX && matchX[1]) return matchX[1];
-  if (matchGeneric && matchGeneric[1]) return matchGeneric[1];
-
-  return 1;
+const applySingleTransformation = (ctx, operation, origin) => {
+  const [name, args] = operation;
+
+  switch (name) {
+    case 'scale': {
+      const [scaleX, scaleY] = args;
+      ctx.scale(scaleX, scaleY, { origin });
+      break;
+    }
+
+    case 'rotate': {
+      ctx.rotate(args, { origin });
+      break;
+    }
+
+    case 'translate': {
+      const [x, y] = args;
+      ctx.translate(x, y, { origin });
+      break;
+    }
+
+    case 'matrix': {
+      ctx.transform(...args);
+      break;
+    }
+
+    default: {
+      console.error(`Transform operation ${name} doesn't supported`);
```

Comment:
does `@react-pdf` have better way for logging errors?

Reply:
Unfortunately not for the moment 😞   What logic can be embedded in a error logging util that a `console.error` does not cover? It might be a good idea to create a new package (`@react-pdf/log` maybe?) in which we can log any sort of events (info, error, warning) and either use that to show in the console or generate reports

Reply:
No, I mean wrap logging with process.env.NODE_ENV like react does, because this seems useless in prod code, but i'm not sure about that

Reply:
Oh got it. No there's nothing like that for the moment 😅